packages/utilities/psammead-styles/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 7.0.0 | [PR#????](https://github.com/bbc/psammead/pull/????) Replace BBC Nassim with BBC Reith Qalam. |
```

Comment:
This is where I go into trouble before - should this be an alpha release?

Reply:
Replacing BBC Nassim with BBC Reith Qalam is a breaking change, so the major version has been incremented. Unless there is a specific reason to add an alpha tag(?), this should be enough.